File: src/test/java/Test/AndroidTests/LoginAndRegTests/LogoutTest.java
package Test.AndroidTests.LoginAndRegTests;

import AppiumSupport.BaseTestClass;

import org.junit.Test;

public class LogoutTest extends BaseTestClass {

    @Test
    public void logoutTest() {
        onboardingInterface.waitThenCloseOnBoardingPage();
        loginInterface.loginAsTester9();
        tapeInterface.closeTooltips();
        bottomToolbarInterface.clickProfile();
        profilePageInterface.waitTooltip();
        profilePageInterface.closeTooltip();
        profilePageInterface.waitSettings();
        profilePageInterface.clickSettings();
        settingsInterface.swipeToLogOut();
        onboardingInterface.waitThenCloseOnBoardingPage();
    }
}

File: src/test/java/Test/AndroidTests/LoginAndRegTests/LogoutTest.kt
package Test.AndroidTests.LoginAndRegTests

import AppiumSupport.BaseTestClass
import Pages.AndroidPages.LoginAndRegistration.User

import org.junit.jupiter.api.Test

class LogoutTest : BaseTestClass() {

    @Test
    fun logoutTest() {
        onboardingInterface.waitThenCloseOnBoardingPage()
        loginInterface.login(User.User9())
        tapeInterface.closeTooltips()
        bottomToolbarInterface.clickProfile()
        profilePageInterface.waitTooltip()
        profilePageInterface.closeTooltip()
        profilePageInterface.waitSettings()
        profilePageInterface.clickSettings()
        settingsInterface.swipeToLogOut()
        onboardingInterface.waitThenCloseOnBoardingPage()
    }
}
